### ISSUE TITLE
User gets a permisison denied error after submitted a pre-approved proposal

### ISSUE THREAD

**Issue by tymees** [DH-IT-Portal-Development/ethics#607]
Needs further diagnosing;

A user reported that they got the infamous 'permission denied' page after submitting a pre-approved proposal. They did get a confirmation e-mail and the application was properly sent for review. So I'm guessing that something went wrong when redirecting to the 'proposal submitted' page. 

Sadly they didn't provide the exact URL of the page they were sent to, so the above is pure speculation.

It's probably another 'we didn't test the pre-approved track' issue....

**Comment by EdoStorm96** [comment]
I was doing some testing surrounding this issue, and what I think happened was, the user pressed the back button on the submit page. This, i guess because the back button is also a submit button, just triggers `start_review` from the view's `form_valid` and updates the status to `Proposal.Statuses.SUBMITTED`. This actually also happens with regular proposals. This then raises this permission denied:

main/views.py, line 384:
```
        if proposal.status >= Proposal.Statuses.SUBMITTED:
            if self.request.user not in commission:
                raise PermissionDenied
```
So basically ... If you press either the '<< save and back', 'return to start of application' or 'submit application' buttons, the proposal always gets submitted. Not just for pre-approved proposals. Any idea on how to fix this? Maybe just make another exception for this in form_buttons.html?

**Comment by tymees** [comment]
No, that cannot be what happened. What you're describing is a bug that's currently only in develop. (And is fixed in #603)

If you look at the code in master (and thus running on the server), you'll see we protect against that happening:
https://github.com/DH-IT-Portal-Development/ethics/blob/master/proposals/views/proposal_views.py#L435

**Comment by miggol** [comment]
I'm closing this as not reproducible. I've tried a variety of things, and discovered a new issue along the way: #663 

However, no success with this issue.

Might have been a freak mail sending accident, which could cause this error upon refresh (I think).
